wscript

```diff
@@ -123,6 +123,7 @@ def configure(ctx):
             libclang = ctx.root.find_node(ctx.options.clang_path)
             if libclang is None:
                  ctx.fatal("%s not found" % ctx.options.clang_path)
+            libclang = ctx.options.clang_path
```

Comment:
The result of `ctx.root.find_node` is the name of the last node, so it returns the name of the library if it is presented, but you need the whole path.